### DISCUSSION TITLE
Live data for Power Flow card

### DISCUSSION THREAD

**Discussion by Paul-G-M** [kellerza/sunsynk#104] [Q&A] [answered]
Is it possible to get live (real-time or near real-time) power data from the Sunsynk integration? If so, how’s it done?
I’d  like to display live data on my home dashboard using the Power Flow Custom card.
I have a 3.6 kW converter connected by RS485 to USB. It all works well into the Energy Dashboard
Thanks folks!

**Comment by kellerza** [comment, marked as answer]
Yes, simply add :now to your sensor - see “sensor modifiers” in the addon docs

**Reply by kellerza** [reply]
https://kellerza.github.io/sunsynk/reference/sensor-modifiers
